libs/shared-web/src/hooks/useFeatureFlags.ts

```diff
@@ -24,9 +24,17 @@ export const useFeatureFlags = (organization?: IOrganizationEntity) => {
 };
 
 export const useFeatureFlag = (key: FeatureFlagsKeysEnum): boolean => {
+  const ldFlagSet = useFlags();
   /** We knowingly break the rule of hooks here to avoid making any LaunchDarkly calls when it is disabled */
-  // eslint-disable-next-line
-  const flagValue = checkShouldUseLaunchDarkly() ? useFlags()[key] : undefined;
+  const flagValue = checkShouldUseLaunchDarkly() ? ldFlagSet[key] : undefined;
+  const isPlaywright = window && (window as any).isPlaywright;
+  if (Object.keys(ldFlagSet).length !== 0 && ldFlagSet[key] === undefined && isPlaywright) {
```

Comment:
```suggestion
  if (flagValue === undefined && isPlaywright) {
```

Reply:
I'll make a comment explaining this, it's not an easy implmentation, this change wouldn't work

Reply:
replaced with 
![image](https://github.com/novuhq/novu/assets/39020298/397b7a3e-97fa-402f-ae9d-b60c042c38ae)